generators/client/templates/angular/src/main/webapp/app/blocks/interceptor/_notification.interceptor.ts

```diff
@@ -18,7 +18,8 @@ export class NotificationInterceptor extends HttpInterceptable {
             let headers = [];
             let i;
             for(i = 0; i < arr.length; i++){
-                if(arr[i][0].endsWith('app-alert') || arr[i][0].endsWith('app-params')) {
+                if(arr[i][0].indexOf('app-alert', arr[i][0].length - 'app-alert'.length) !== -1 ||
```

Comment:
no need to change here as this is typescript. You need to change this in the templates/angularjs folder inside the client which is in JS and will be affected